src/main/java/com/teamscale/buildbreaker/exceptions/CommitCouldNotBeResolvedException.java

```diff
@@ -1,7 +0,0 @@
-package com.teamscale.buildbreaker.exceptions;
-
-public class CommitCouldNotBeResolvedException extends BuildBreakerExceptionBase {
```

Comment:
Not really deleted, but changed. This is not a BuildBreakerException now because it's thrown by the client an the client should be independent of the command line.